docs/fuse.md

```diff
@@ -26,6 +28,12 @@ go get github.com/jbenet/go-fuse-version/fuse-version
 
 If you run into any problems installing FUSE or mounting IPFS, hop on IRC and speak with us, or if you figure something new out, please add to this document!
 
+You may also have to add your username to the `operator` group:
```

Comment:
haha fair. It works for me without (am not an operator, am in `staff` though). But i did have to:

```
sudo chown jbenet /ipfs
sudo chown jbenet /ipns
```